R/R.r

```diff
@@ -33,21 +33,24 @@ RCMD <- function(cmd, options, path = tempdir(), env_vars = NULL, ...) {
 #' between the current R session and the new session, and to ensure that
 #' everything behaves the same across systems. It also suppresses a common
 #' warning on windows, and sets \code{NOT_CRAN} so you can tell that your
-#' code is not running on CRAN.
+#' code is not running on CRAN. If \code{NOT_CRAN} has been set externally, it
+#' is not overwritten.
 #'
 #' @keywords internal
 #' @return a named character vector
 #' @export
 r_env_vars <- function() {
-  c("R_LIBS" = paste(.libPaths(), collapse = .Platform$path.sep),
+  vars <- c("R_LIBS" = paste(.libPaths(), collapse = .Platform$path.sep),
     "CYGWIN" = "nodosfilewarning",
     # When R CMD check runs tests, it sets R_TESTS. When the tests
     # themselves run R CMD xxxx, as is the case with the tests in
     # devtools, having R_TESTS set causes errors because it confuses
     # the R subprocesses. Un-setting it here avoids those problems.
     "R_TESTS" = "",
-    "NOT_CRAN" = "true",
     "TAR" = auto_tar())
+
+  if(is.na(Sys.getenv("NOT_CRAN", unset = NA))) c(vars, "NOT_CRAN" = "true")
```

Comment:
Could you please use `{}` here, and indent? (e.g. as described in the curly braces section of http://adv-r.had.co.nz/Style.html)